release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -629,6 +636,20 @@ For further information about enabling more control over the C-states, see xref:
 ==== Support for provisioning IPv6 spoke clusters from dual-stack hub clusters
 With this update, you can provision IPv6 address spoke clusters from dual-stack hub clusters. In a zero touch provisioning (ZTP) environment, the HTTP server on the hub cluster that hosts the boot ISO now listens on both IPv4 and IPv6 networks. The provisioning service also checks the baseboard management controller (BMC) address scheme on the target spoke cluster and provides a matching URL for the installation media. These updates offer the ability to provision single-stack, IPv6 spoke clusters from a dual-stack hub cluster.
 
+[id="ocp-4-14-nw-shiftstack-dual-stack"]
```

Comment:
It might be worth to point the user to the dual-stack docs for more information like it was done for CPMS release notes.
https://github.com/openshift/openshift-docs/pull/63935

Reply:
BTW, the dual-stack docs gives additional context on what scenarios are supported and what are not.